packages/core/ios/Sources/DdRumImplementation.swift

```diff
@@ -181,6 +181,13 @@ public class DdRumImplementation: NSObject {
         }
         resolve(nil)
     }
+    
+    @objc
+    public func getCurrentSessionId(_ resolve: @escaping (Any?) -> Void, reject: RCTPromiseRejectBlock) -> Void {
```

Comment:
As I have mentioned in the PR description, I had to use `_ resolve: @escaping (Any?) -> Void` instead of `RCTPromiseResolveBlock` in order for the Swift Bridging Header to be generated correctly.